Tests_Turorials/how.to.contribute.md

```diff
@@ -0,0 +1,88 @@
+# How to contribute to test development
+
+## Where to create a test file
+
+PYLEECAN classes are generated automatically, all the classes are built in the same way. 
+This fact enables us to test all the classes with one test file which can be find in Tests/Classes. 
+So it's useless to develop tests for the class itself but it's still use to develop the tests for the methods defined in the Class Creation.
```

Comment:
> 
> 
> That's not entirely true. Rather it would be useful to develop tests for the classes, since there may be issues inheritent due to the class generator that are not issues of the generators methods itself (e.g. [the Dict Copy Issue](https://github.com/Eomys/pyleecan/pull/183)). But it would be inconsistent to have these tests in different test files. Better, these test should extend Tests/Classes/test_classes.py instead.

Yes you're right. Thanks for the feedback ! I'm changing it.